MyUnitTests/MyTargetClassTests.cs

```diff
@@ -6,23 +6,47 @@ namespace MyUnitTests
     public class MyTargetClassTests
     {
         [Fact]
-        public void VerifySummation()
+        public void Summary_WhenTwoIntegersAreProvided_ShouldReturnTotalAsInteger()
         {
-            const int intFirstNumber = 15;
-            const int intsecondNumber = 10;
-            var objTargetClass = new MyTargetClass();
-            var intResult = objTargetClass.AddNumbers(intFirstNumber, intsecondNumber);
-            Assert.Equal(15 + 10, intResult);
+            // Arrange
+            MyTargetClass summtest = new MyTargetClass();
+            // Act
+            int actual = summtest.AddNumbers(15,10);
+            //Assert
+            Assert.Equal(25, actual);
         }
 
         [Fact]
-        public void VerifySubtraction()
+        public void Subtract_WhenTwoIntegersAreProvided_ShouldReturnTotalAsInteger()
         {
-            const int intFirstNumber = 15;
-            const int intsecondNumber = 10;
-            var objTargetClass = new MyTargetClass();
-            var intResult = objTargetClass.SubractNumbers(intFirstNumber, intsecondNumber);
-            Assert.Equal(15 - 10, intResult);
+            // Arrange
+            MyTargetClass subtracttest = new MyTargetClass();
+            // Act
+            int actual = subtracttest.SubractNumbers(15,10);
```

Comment:
Please put a space after 15,
Also please define class level constants for reused integers. (for 15 and 10)